skietbaan_fe/src/components/ScoreCapture.js

```diff
@@ -175,317 +199,291 @@ export default class search extends Component {
+        <div className="position-relative">
+          <div className={stateOne ? "page-content-video" : "page-content"}>
+            <div className={stateOne ? "hidden" : ""}>
+
+              <div className="label-score">
+                <div className="centre-label">
+                  <label className="scorelabel">Type in score</label>
+                </div>
+                <div className="input-container">
+                  <input type="number" id="scoreInput" min="0" step="1" name="score" className="score"
+                    onChange={this.handleScore}></input>
+                  <div className={this.state.validScore ? "hidden" : "invalidScore"}>Enter Valid Score</div>
+                </div>
 
-            <div className="label-score">
-              <div className="centre-label">
-                <label className="scorelabel">Type in score</label>
               </div>
-              <div className="input-container">
-                <input type="number" id="scoreInput" min="0" step="1" name="score" className="score"
-                  onChange={this.handleScore}></input>
-                <div className={this.state.validScore ? "hidden" : "invalidScore"}>Enter Score</div>
+              <div className="centre-label">
+                <label className="label-competition">Select Competition</label>
               </div>
+              <div className="competition-container">
+                {competitionItem}
 
-            </div>
-            <div className="centre-label">
-              <label className="label-competition">Select Competition</label>
-            </div>
-            <div className="competition-container">
-              {competitionItem}
-
-            </div>
-            <div className={this.state.validCompetition ? "hidden" : "invalidComp"}>Select Competition</div>
-            <div className={this.state.scoreSaved ? "sucess-container" : "hidden"}>
-              <div className="success"> Score Saved successfully </div>
-            </div>
-          </div>
-          <div className="submit-container" id="HideWhenAddingScore">
-            <div className={this.state.ImageTaken || this.state.showCamera
-              ? "hidden" : "submit-button-elements"}>
-              <div className="button-hover">
-                <img src={cameraGray}
-                  id="btnScoreCapture" className="btnScoreCapture"
-                  onClick={() => this.CameraClicked()} alt=''></img>
               </div>
-              <label className="labelIcon">Capture score</label>
-            </div>
-            <div className={(this.state.showCamera && !this.state.ImageTaken)
-              || this.state.ImageTaken ? "hidden" : "submit-button-elements"}>
-              <div className="button-hover ">
-                <img src={graySubmit} onClick={() => this.GetLocation()}
-                  className="button-that-submits" alt=''></img>
+              <div className={this.state.scoreSaved ? "sucess-container" : "hidden"}>
```

Comment:
inconsistent naming convention fro css classes

Reply:
what? its the same

Reply:
Apologise confused state var with css classname